src/verifiers/documentStatus/tokenRegistry/ethereumPolTokenRegistryStatus.test.ts

```
@@ -0,0 +1,82 @@
+    it("should create an infura provider for 'pol' (normalised to matic internally)", () => {
+      expect(() =>
+        getDefaultProvider({ network: "pol" })
+      ).not.toThrow();
+    });
+  });
+
+  describe("verificationBuilder with network: 'pol'", () => {
+    it("should create a verifier without throwing", () => {
+      expect(() =>
+        verificationBuilder(openAttestationVerifiers, { network: "pol" })
+      ).not.toThrow();
```

ESLint

[error] Replace `⏎········verificationBuilder(openAttestationVerifiers,·{·network:·"pol"·})⏎······` with `·verificationBuilder(openAttestationVerifiers,·{·network:·"pol"·})`

(prettier/prettier)